src/main/twirl/gitbucket/core/pulls/menu.scala.html

```diff
@@ -122,5 +122,7 @@ <h1 class="body-title">
 </script>
 
 @countConversation(comments: Seq[Comment]) = @{
-  comments.count(c => c.isInstanceOf[CommitComments] || c.asInstanceOf[IssueComment].action.endsWith("comment"))
+  comments.filter(c => c.isInstanceOf[CommitComments]).map(c => c.asInstanceOf[CommitComments].comments.size).sum +
+    comments.filter(c => c.isInstanceOf[IssueComment]).map(c => c.asInstanceOf[IssueComment]).count(c => c.action.endsWith("comment"))
```

Comment:
You can do it more easily using `collect` as follows:

```scala
comments.collect { 
  case c: CommitComments => c.comments.size
  case c: IssueComment if c.action.endsWith("comment") => 1 
}.sum
```

Reply:
Thanks for your advice! It is very easy !!
I will change and push at tonight.

Reply:
@takezoe I've changed and pushed.